chapter_3/.rubocop.yml

```diff
@@ -0,0 +1,3 @@
+AllCops:
```

Comment:
You don't need a .rubocop.yml file for each homework directory. If you run guard within the parent directory (lrthw_exercises), guard should find the right .rubocop.yml